gigasecond/gigasecond.go

```diff
@@ -0,0 +1,17 @@
+// This is a "stub" file.  It's a little start on your solution.
+// It's not a complete solution though; you have to write some code.
+
+// Exercism package provinding a function adding a gigasecond to an
+// input date
+package gigasecond
+
+import "time"
+
+// Adds a gigasecond to an input time
```

Comment:
You are violating the recommendations for writing comments [here][1] by not starting with the method name first and not ending with a dot.

[1]: https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Did no know about that, will keep in mind.